.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Write a sentence or two here describing what this PR includes along with any additional information that could help reviewers.
+
+If this PR adds a new page of documentation, be sure to edit the `README.md` file to include a link so that it can be found.
+
+If this PR fixes an [issue](https://github.com/openshift/openshift-origin-design/issues), include a [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close that issue once this pull request is merged. Example: `Closes #123`
+
+Remove any of these team mentions to prevent them from being notified:
+@openshift/team-ux-leads, @openshift/team-ux-review, @openshift/team-kni-ux
```

Comment:
would you mind adding @openshift/team-devconsole-ux to this list as well? Maybe some guidance around which to choose, too? All should go to team-ux-leads, core items would go to team-ux-review, kni/kubevirt items would go to team-kni-ux, and Dev Console items would go to team-devconsole-ux.

Reply:
Tweaked this a bit, marking the last 3 optional with a brief description of who they are.

Reply:
Should "Core UX team" be "Admin Console UX team" to more clearly differentiate from the "Dev Console UX team"?

Reply:
Yes, "Admin Console UX team" sounds much more clear to me :)

Reply:
Changed, thanks!